met-web/src/components/common/dateHelper.tsx

```diff
@@ -11,7 +11,7 @@ const formatInTimeZone = (date: string, fmt: string, tz: string) =>
 
 export const formatDate = (d: string, formatString = 'yyyy-MM-dd') => {
     if (d) {
```

Comment:
If along the way we can change this variable name from d to date it would be great.